tests/test_KD_Lib.py

```diff
@@ -7,11 +7,17 @@
 from KD_Lib.TAKD.main import main_TAKD
 from KD_Lib import KD_Lib
 from KD_Lib.original.mnist import mnist
-
+from KD_Lib.models.resnet import ResNet18, ResNet34, ResNet50, ResNet101, ResNet152
 
 def test_mnist():
     mnist(epochs=0)
 
+def test_resnet():
+    ResNet18([32, 32, 64, 64, 128])
+    ResNet34([32, 32, 64, 64, 128])
+    ResNet50([32, 32, 64, 64, 128])
+    ResNet101([32, 32, 64, 64, 128])
+    ResNet152([32, 32, 64, 64, 128])
```

Comment:
Make these parameters smaller, something like `[4,4,8,8,16]` because they might cause memory issues for travis later when we add more tests. Also make a variable for this list and pass It to all the resnet models.

Reply:
Ok sure. I'll get that done